extensions/ql-vscode/src/view/results/ResultsApp.tsx

```diff
@@ -173,7 +173,7 @@ export function ResultsApp() {
               },
               selectedTable: tableName,
             },
-            origResultsPaths: undefined as any, // FIXME: Not used for interpreted, refactor so this is not needed
+            origResultsPaths: undefined as unknown as ResultsPaths, // FIXME: Not used for interpreted, refactor so this is not needed
```

Comment:
This whole situation in `ResultsApp.tsx` is a bit unfortunate. There are a few arguments where we pass `undefined` or other "fake" values. I briefly looked at trying to do the refactoring that this comment asks for but I found this area of code very confusing. For now I think doing `as unknown as ResultsPaths` gets the same effect and isn't any worse than `as any`. 🤷🏼